manifests/h/HamRadioDeluxe/HamRadioDeluxe/v6.8.0.372/HamRadioDeluxe.HamRadioDeluxe.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: HamRadioDeluxe.HamRadioDeluxe
+PackageVersion: v6.8.0.372

```

```suggestion
PackageVersion: 6.8.0.372
```
This should match control panel exactly. After changing this, the folder name will need to be updated.